src/CacheManager.Serialization.Bond/project.json

```diff
@@ -0,0 +1,53 @@
+{
+  "version": "0.9.0-*",
+  "description": "CacheManager extension package providing Bond serialization for distributed caches. CacheManager is an open source caching abstraction layer for .NET written in C#. It supports various cache providers and implements many advanced features, the Bond serializer can be used in place of the default Binary Serializer",
+  "packOptions": {
+    "licenseUrl": "https://github.com/MichaCo/CacheManager/blob/master/LICENSE",
+    "projectUrl": "http://www.cachemanager.net",
+    "repository": {
+      "type": "git",
+      "url": "https://github.com/MichaCo/CacheManager"
+    },
+    "summary": "CacheManager extension package providing Bond serialization for distributed caches.",
+    "tags": [ "Caching", "Cache", "CacheManager", "Distributed Cache", "StackExchange Redis", "Azure AppFabric", "Memcached" ]
+  },
+  "authors": [ "loneshark99" ],
+  "buildOptions": {
+    "allowUnsafe": false,
+    "languageVersion": "csharp6",
+    "warningsAsErrors": true
+  },
+  "copyright": "Copyright (c) 2015 MichaConrad",
+  "configurations": {
+    "Debug": {
+      "buildOptions": {
+        "define": [ "DEBUG" ]
+      }
+    },
+    "Release": {
+      "buildOptions": {
+        "define": [ "RELEASE" ],
+        "optimize": true
+      }
+    }
+  },
+  "dependencies": {
+    "Bond.CSharp": "4.2.1",
+    "CacheManager.Core": "0.9.0-*",
+    "NETStandard.Library": "1.6.0"
+  },
+
+  "frameworks": {
+    "netstandard1.3": {
+      "imports": "dnxcore50"
```

Comment:
Sure I will update this and add tests today. Thank you for considering it.

On Sunday, August 21, 2016, MichaC <notifications@github.com> wrote:

> In src/CacheManager.Serialization.Bond/project.json
> <https://github.com/MichaCo/CacheManager/pull/83#discussion_r75595732>:
>
> > +    "Release": {
> > +      "buildOptions": {
> > +        "define": [ "RELEASE" ],
> > +        "optimize": true
> > +      }
> > +    }
> > +  },
> > +  "dependencies": {
> > +    "Bond.CSharp": "4.2.1",
> > +    "CacheManager.Core": "0.9.0-*",
> > +    "NETStandard.Library": "1.6.0"
> > +  },
> > +
> > +  "frameworks": {
> > +    "netstandard1.3": {
> > +      "imports": "dnxcore50"
>
> "imports" should not be needed
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/MichaCo/CacheManager/pull/83/files/5cda0606fddece283df22ac11e6dcd44b1d4d761#r75595732>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHh_AKLh9ImQ3txOtWSfMJ9jrMPrDq45ks5qiFr0gaJpZM4JovRa>
> .
>